src/guides/light_push_send_messages.md

```diff
@@ -0,0 +1,85 @@
+# Send Messages Using Waku Light Push
+
+Waku Light Push enables a client to receive a confirmation when sending a message.
+
+The Waku Relay protocol sends messages to connected peers but does not provide any information on whether said peers have received messages.
+This can be an issue when facing potential connectivity issues.
```

Comment:
For example, when peer count is lower than <some number> (gossipsub low parameter probably, 4?)